test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskTests.cs

```diff
@@ -765,6 +765,72 @@ public Task MsbuildRestore_WithStaticGraphRestore_MessageLoggedAtDefaultVerbosit
             return Task.CompletedTask;
         }
 
+        [PlatformFact(Platform.Windows)]
+        public async Task MsbuildRestore_StaticGraphEvaluation_HandlesInvalidProjectFileExceptionn()
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+
+                var net461 = NuGetFramework.Parse("net461");
+
+                var projectA = new SimpleTestProjectContext("a", ProjectStyle.PackageReference, pathContext.SolutionRoot);
+
+                var projectB = new SimpleTestProjectContext("b", ProjectStyle.PackageReference, pathContext.SolutionRoot);
+                var projectC = new SimpleTestProjectContext("c", ProjectStyle.PackageReference, pathContext.SolutionRoot);
+
+                var projectAFrameworkContext = new SimpleTestProjectFrameworkContext(net461);
+
+                projectAFrameworkContext.ProjectReferences.Add(projectB);
+                projectAFrameworkContext.ProjectReferences.Add(projectC);
+
+                var packageX = new SimpleTestPackageContext()
+                {
+                    Id = "x",
+                    Version = "1.0.0"
+                };
+                projectA.Frameworks.Add(projectAFrameworkContext);
+
+                packageX.Files.Clear();
+                projectA.AddPackageToAllFrameworks(packageX);
+                packageX.AddFile("lib/net461/a.dll");
+
+                solution.Projects.Add(projectA);
+                solution.Create(pathContext.SolutionRoot);
+
+                var configAPath = Path.Combine(Path.GetDirectoryName(projectA.ProjectPath), "NuGet.Config");
+                var configText =
+$@"<?xml version=""1.0"" encoding=""utf-8""?>
+<configuration>
+    <packageSources>
+        <add key=""LocalSource"" value=""{pathContext.PackageSource}"" />
+    </packageSources>
+</configuration>";
+                using (var writer = new StreamWriter(configAPath))
+                {
+                    writer.Write(configText);
+                }
+
+                await SimpleTestPackageUtility.CreateFolderFeedV3Async(
+                    pathContext.PackageSource,
+                    packageX);
```

Comment:
`packageX` doesn't seem to be used in any of the asserts, so I believe it can be removed from the arrange and save a little time not needing to create the nupkg.

Reply:
Without a PackageReference I get this:

```
Nothing to do. None of the projects specified contain packages to restore.
```

But it does look like I can get rid of the package creation since its not actually used